virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc

```diff
@@ -0,0 +1,27 @@
+[id="virt-reviewing-vm-dashboard"]
+= Reviewing resource usage by virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-reviewing-vm-dashboard
+
+toc::[]
+
+Dashboards in the {ProductName} web console provide visual representations of cluster metrics to help you to quickly understand the state of your cluster. Dashboards belong to the xref:../../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[monitoring stack] that provides monitoring for core platform components.
```

Comment:
`{ProductName}` is showing up verbatim in the preview build, which I don't see in `modules/virt-document-attributes.adoc`. Should this maybe be either `{product-title}` or `{VirtProductName}`?

Reply:
I believe Lavanya meant this to be the variable for the OCP web console. Changing to {product-title}.
cc: @lmandavi